### PR TITLE
Add 'Content Fleet GmbH' (community contribution)

### PR DESCRIPTION
**Pull request by datenanfragen-community-edits**
This suggestion was submitted through the website.

**[Edit](https://company-json.datenanfragen.de/#!doc=%7B%22slug%22%3A%22contentfleet%22%2C%22relevant-countries%22%3A%5B%22de%22%5D%2C%22name%22%3A%22Content%20Fleet%20GmbH%22%2C%22runs%22%3A%5B%22einfachtierisch.de%22%5D%2C%22address%22%3A%22Kehrwieder%208%5Cn20457%20Hamburg%22%2C%22email%22%3A%22datenschutz%40contentfleet.com%22%2C%22web%22%3A%22https%3A%2F%2Fwww.contentfleet.com%22%2C%22sources%22%3A%5B%22https%3A%2F%2Feinfachtierisch.de%2Fp%2Fimpressum%22%2C%22https%3A%2F%2Feinfachtierisch.de%2Fp%2Fdatenschutz%22%2C%22https%3A%2F%2Fgithub.com%2Fdatenanfragen%2Fdata%2Fpull%2F1340%22%5D%2C%22suggested-transport-medium%22%3A%22email%22%2C%22quality%22%3A%22verified%22%7D)**

### CODE REVIEW

**Comment by WebworkrNet** [comment]
I have opened this issue via [datenanfragen.de](https://www.datenanfragen.de).
You can contact me if you have any questions.

File: companies/contentfleet.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "contentfleet",
+    "relevant-countries": [
+        "de"
+    ],
+    "name": "Content Fleet GmbH",
+    "runs": [
+        "einfachtierisch.de"
+    ],
+    "address": "Kehrwieder 8\n20457 Hamburg",
```
Comment (moriarty-jim) [inline companies/contentfleet.json:10]:
```suggestion
    "address": "c/o Ströer Digital Media GmbH\nPlatz der Einheit 1\n60327 Frankfurt am Main\nDeutschland",
```
address of DPO, see [homepage](https://einfachtierisch.de/p/datenschutz)

File: companies/contentfleet.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "email": "datenschutz@contentfleet.com",
```
Comment (moriarty-jim) [inline companies/contentfleet.json:11]:
```suggestion
    "email": "datenschutz@contentfleet.com",
    "phone": "+49 30 12085507",
    "fax": "+49 40 600800619",
```
added phone and generic fax

**Review by moriarty-jim — CHANGES_REQUESTED** [review]
updated address, added phone & fax

File: companies/contentfleet.json (hunk 2 above)
Comment (baltpeter) [reply companies/contentfleet.json:11]: Not exactly important but I prefer to keep them in the same order as in the schema.

```suggestion
    "phone": "+49 30 12085507",
    "fax": "+49 40 600800619",
    "email": "datenschutz@contentfleet.com",
```

**Review by baltpeter — APPROVED** [review]
Thanks, @WebworkrNet and @moriarty-jim!

**Pull request merged**